### PR TITLE
Fix navigation from feedback

### PR DESCRIPTION
**Pull request by DMurf**
Fix Navigation issues raised during user feedback. 

This merge will add in breadcrumbs on all admin pages and dashboard pages to enable easier movement through the system.

### CODE REVIEW

**Comment by DMurf** [comment]
Current tests passing

File: Introspective.Web/Views/NikoNikoEntry/List.cshtml [hunk 1]
```diff
@@ -1,61 +1,77 @@
 ﻿@model IEnumerable<Introspective.ViewModels.NikoNikoEntryViewModel>
 
 @{
-    ViewBag.Title = "List NikoNikoEntrys";
+    ViewBag.Title = "List Niko Niko Entrys";
```
Comment (ynxe) [inline Introspective.Web/Views/NikoNikoEntry/List.cshtml]: Should this be "Niko Niko Entries"?

There's a few instances of this spelling through these pages

**Review by ynxe — APPROVED** [review]
LGTM, very thorough.
Given the large number of pages it be worth setting up a helper function to make the breadcrumbs generic and automatically populated via page templating.

Caught a spelling error, duplicated in a couple of places. Please tweak that before merging down.

🫡

File: Introspective.Web/Views/NikoNikoEntry/List.cshtml (hunk 1 above)
Comment (DMurf) [reply Introspective.Web/Views/NikoNikoEntry/List.cshtml]: Perfect, changed it. Thanks!

**Comment by DMurf** [comment]
PR complete, merging.

**Pull request merged**